api/src/main/java/org/eclipse/microprofile/config/spi/ConfigBuilder.java

```diff
@@ -65,6 +65,14 @@
      */
     ConfigBuilder addDiscoveredConverters();
 
+    /**
+     * Add all interceptors which can be <a href="ConfigSourceInterceptor.html#discovery">discovered</a> from
+     * this configuration builder's {@linkplain #forClassLoader(ClassLoader) class loader}.
+     *
+     * @return this configuration builder instance
+     */
+    ConfigBuilder addDiscoveredInterceptors();
```

Comment:
Can't see the ConfigSourceInterceptor type in master nor in this PR

Reply:
Yes, this is from an old proposal and forgot to remove this. I'll remove it.